### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by kfoley123**

### CODE REVIEW

File: src/Pages/Projects/Projects.js [hunk 1]
```diff
@@ -1,9 +1,42 @@
 import React from "react";
+import HugeProject from "../../Components/HugeProject/HugeProject";
+import "./Projects.css";
 
 export default function Projects() {
+    let projectsData = [
+        {
+            title: "XCOMMITEE",
```
Comment (oakeef) [inline src/Pages/Projects/Projects.js:8]:
```suggestion
            title: "XCOMMITTEE",
```

File: src/Components/HugeProject/HugeProject.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    width: 510px;
+    overflow-y: scroll;
+}
+
+.hugeProjectDescription {
+    color: grey;
+    font-size: 18px;
+    font-weight: 500;
+    padding: 0 20px 20px 20px;
+    overflow-y: scroll;
```
Comment (oakeef) [inline src/Components/HugeProject/HugeProject.css:41]: I tihnk we should bail on the scrolling thing and write shorter descriptions. Then link each project to it's own project subpage where we can flesh them out with images and more text

File: src/Pages/Projects/Projects.js [hunk 3]
```diff
@@ -1,9 +1,42 @@
 import React from "react";
+import HugeProject from "../../Components/HugeProject/HugeProject";
+import "./Projects.css";
 
 export default function Projects() {
+    let projectsData = [
+        {
+            title: "X-COMMITEE",
```
Comment (oakeef) [inline src/Pages/Projects/Projects.js:8]:
```suggestion
            title: "X-COMMITTEE",
```

File: src/Components/HugeProject/HugeProject.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    align-items: center;
+    background-color: white;
+    border-radius: 20px;
+    box-shadow: 0 0px 20px 0 rgba(79, 73, 63, 0.5);
+    display: flex;
+    flex-direction: column;
+    height: 550px;
+    margin: 10px;
+    position: relative;
+    width: 550px;
```
Comment (kfoley123) [inline src/Components/HugeProject/HugeProject.css:11]:
```suggestion
    width: 551px;
```

**Review by oakeef — APPROVED** [review]

**Pull request merged**